server/command/main.go

```diff
@@ -25,7 +25,8 @@ func (s *service) handleMain(in *params) (*model.CommandResponse, error) {
 		"debug-install-builtin": s.executeDebugInstallBuiltinHello,
 		"debug-install-http":    s.executeDebugInstallHTTPHello,
 		"debug-install-aws":     s.executeDebugInstallAWSHello,
-		"provision":             s.executeProvision,
+		"provision-aws":         s.executeProvisionAWSApp,
+		"install-aws":           s.executeInstallAWSApp,
```

Comment:
Whenever we add a command/other entry point, it would be helpful to document (README, some other place) info like:
- what does it do, who is supposed to run it, context needed, constraints, usage scenarios, etc.